product-service/src/functions/getProductsList/handler.ts

```diff
@@ -0,0 +1,25 @@
+import { APIGatewayProxyResult } from 'aws-lambda/trigger/api-gateway-proxy';
+import { middyfy } from '@libs/lambda';
+import { type ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { IDatabaseService } from 'src/services/database-service';
+import { okResponse } from 'src/responses';
+import { databaseService } from 'src/dependencies';
+
+const getProductsList = (databaseService: IDatabaseService): ValidatedEventAPIGatewayProxyEvent<APIGatewayProxyResult> => async (event) => {
+  console.log(event);
+
+  const productDocuments = await databaseService.scanProducts();
```

Comment:
What if you get a DB error in lambda? In this case, lambda can retry a few times, which we don't want. You would need to put everything here in try-catch block and in case of an error return with an error response and log error to the console (not return it to the client)